apps/project-renderer/src/components/app/index.js

```diff
@@ -67,6 +68,9 @@ const App = ( {
 			} );
 	}, [ projectCode, preview ] );
 
+	const { stopSubmit } = useDispatch( STORE_NAME );
+	const formName = `f-${ projectCode }`;
```

Comment:
If we go with the application level approach (see my other comment below), what do you think about passing `startSubmit` and `stopSubmit` to the submit handler from the `<Form>` component rather than accessing the form store directly here?

So we'd end up with something like `const handleSubmit = ( { data, startSubmit, stopSubmit } ) => ...`

Reply:
I'm reverting this and handling the submission status inside the Form component.